src/core/MOM.F90

```diff
@@ -1729,9 +1730,12 @@ subroutine initialize_MOM(Time, Time_init, param_file, dirs, CS, restart_CSp, &
   real, allocatable         :: v_in(:,:,:) ! Initial meridional velocities [L T-1 ~> m s-1]
   real, allocatable         :: h_in(:,:,:) ! Initial layer thicknesses [H ~> m or kg m-2]
   real, allocatable, target :: frac_shelf_in(:,:) ! Initial fraction of the total cell area occupied
-                                           ! by an ice shelf [nondim]
+                                                  ! by an ice shelf [nondim]
+  real, allocatable, target :: mass_shelf_in(:,:) ! Initial mass of ice shelf contained within a grid cell
+                                                  ! [RZ ~> kg m-2]
```

Comment:
The standard syntax here would be `! [R Z ~> kg m-2]`, like on line 283, to allow this comment to be used in generating the lists of unit conversion factors.

Reply:
There are still a number of new lines that do not include the space between the R and Z in the unit conversion factors in the comments (i.e., are `[RZ ~> kg m-2]` instead of `[R Z ~> kg m-2]`.  Given that the list of the units used in the MOM6 code are going to be semi-automatically generated by grepping, sedding, and sorting these unit descriptions from the code, and that the white space between these units is what is used to detect the separately scaled R and Z units, this space is significant.  Please revise all such instances in the new code, so as not to break our expanded abilities to document and detect dimensional inconsistencies, and contained in MOM6 PR #49.